zeebe/gateway-rest/src/main/java/io/camunda/zeebe/gateway/rest/controller/UserTaskController.java

```diff
@@ -120,4 +130,34 @@ private static ProblemDetail mapRejectionToProblem(final BrokerRejection rejecti
         }
     };
   }
+
+  @PostMapping(
+      path = "/user-tasks/search",
+      produces = {MediaType.APPLICATION_JSON_VALUE, MediaType.APPLICATION_PROBLEM_JSON_VALUE},
+      consumes = MediaType.APPLICATION_JSON_VALUE)
+  public ResponseEntity<UserTaskSearchQueryResponse> searchUserTasks(
+      @RequestBody final UserTaskSearchQueryRequest query) {
```

Comment:
❓ This makes sending a request body mandatory. Can I also search without any conditions, i.e. without sending a request body?

Reply:
In fact, this was not supporting empty requests, I just updated the code on https://github.com/camunda/camunda/pull/19625/commits/c116ac701985659ca0fd9f45ed414b727b5861f7 , so the request body is no longer required